### ISSUE TITLE
DB access with Adminer

### ISSUE THREAD

**Issue by superkekko** [Atheos/Atheos#228]
Hi I've developed a simple plugin to access DB using Aminer (https://github.com/superkekko/atheos-adminer) 

![immagine](https://user-images.githubusercontent.com/25342565/166635854-2f44870d-2041-43b8-9e31-b34194c2ddf8.png)

Clicking on the icon a new page will be open with Adminer.

**Comment by superkekko** [comment]
Yeah, that is a test, i removed it because is unecessary (Adminer ask the password when you open it). I've added (as a security the code
`require_once("../../common.php");
Common::checkSession(); // Verify Session or Key`
to enable the access only for registered users.

If you would like to edit it feel free to go on. I think that a great thing will be have Adminer in a tab instead in a new window but I don't know ho to do it.

**Comment by HLSiira** [comment]
Weirdly enough, I actually just figured out how to do that myself a few days ago:
```javascript
var path = atheos.path + 'plugins/Adminer/adminer.php';
window.open(path, '_self');
```

It's something like that anyway.

How well does adding that line in the PHP file work? It seems like it would be effective but I've never thought to try it out.

**Comment by superkekko** [comment]
Maybe I'm not explaining well, I would like to open Adminer in a Atheos tab (here)

![immagine](https://user-images.githubusercontent.com/25342565/166956044-d3a93dd9-3fb8-45fb-a858-0cd09c692a97.png)

For the restrinction in php ti works pretty well and it's easy (I've testest and it seem secure). I'm not happy to modify the Adminer file but is the only way to disable public access.
